types/types.go

```diff
@@ -1546,3 +1560,69 @@ func (r RecordType) Types() []Type {
 func (r RecordType) Concat(other RecordType) RecordType {
 	return RecordType{types: append(r.Types(), other.Types()...)}
 }
+
+func (d Date) ToTimeString() string {
+	date := civil.Date{Year: 1970, Month: time.January, Day: 1}
+	date = date.AddDays(int(d))
+	return date.String()
```

Comment:
what's the benefit to this over just using `time` like we were doing before?

Reply:
Adding days to time.t can be messed up by daylight savings time.  Civil date's base type is days so it isn't affected.